docs/src/tutorial.md

```diff
@@ -65,15 +70,27 @@ function jump_example(n::Int)
 end
 
 jumpnlp_example = jump_example(n)
-pqn_jumpnlp = PartiallySeparableNLPModel(jumpnlp_example)
+pqn_jumpnlp = PSNLPModel(jumpnlp_example)
 
 fx = NLPModels.obj(pqn_jumpnlp, x) # compute the obective function
 gx = NLPModels.grad(pqn_jumpnlp, x) # compute the gradient
 ```
 In version v0.2.0, [`ManualNLPModel`](https://github.com/JuliaSmoothOptimizers/ManualNLPModels.jl)s will be supported.
 
-## A partitioned quasi-Newton `NLPModel`
-When defining a `PartiallySeparableNLPModel`, you allocate storage for partitioned quasi-Newton updates, which are implemented in `PartitionedStructures.jl`.
+## Partitioned quasi-Newton `NLPModel`s
+A model deriving from `AbstractPQNNLPModel<:AbstractPartiallySeparableNLPModel` allocates storage required for partitioned quasi-Newton updates, which are implemented in `PartitionedStructures.jl` (see the [PartitionedStructures.jl tutorial](https://juliasmoothoptimizers.github.io/PartitionedStructures.jl/dev/tutorial/) for more details).
+There are several variants:
+* 'PBFGSNLPModel': every element-Hessian approximation is updated with BFGS;
+* 'PSR1NLPModel': every element-Hessian approximation is updated with SR1;
+* 'PSENLPModel': every element-Hessian approximation is updated with BFGS if the curvature condition holds, or withSR1 otherwise;
```

Comment:
```suggestion
* 'PSENLPModel': every element-Hessian approximation is updated with BFGS if the curvature condition holds, or with SR1 otherwise;
```